src/genie/libs/parser/iosxe/show_interface.py

```diff
@@ -3524,7 +3529,10 @@ def cli(self, interface, output=None):
 
         # number of lanes 1
         p2 = re.compile(r'^number +of +lanes +(?P<lanes>[\d]+)$')
-
+        
+        #transceiver info
+        p3 = re.compile(r'^(?P<port>\S+)\s+(?P<value>[\-0-9][0-9\.]+[0-9])\s+(?P<HAT>[\-0-9][0-9\.]+[0-9])\s+(?P<HWT>[\-0-9][0-9\.]+[0-9])\s+(?P<LWT>[\-0-9][0-9\.]+[0-9])\s+(?P<LAT>[\-0-9][0-9\.]+[0-9])$')
```

Comment:
For traceability please add the complete line of output that p3 is parsing in place of transceiver info